### ISSUE TITLE
[VirtualEnv] sys.prefix and base_prefix return the same value in virtualenv  environment

### ISSUE THREAD

**Issue by horstmannmat** [morpheus65535/bazarr#1535]
I still get error when  using virtualenv.

sys.prefix and base_prefix will return the same value, even when inside a  virtualenv  environment .

https://github.com/morpheus65535/bazarr/blob/c66d5662b43fc29068335b26929824e0ec01a219/bazarr/init.py#L48-L49

real_prefix will return the correct path to python when using virtual env or None when using the default python:

I just created a python test to print the values, in the first test I ran with the default python binary.
The second test was using the virtualenv environment
```
horstmann@linus:~$ python test.py 
base_prefix:  /usr
real_prefix:  None
sys.prefix:  /usr
False

(virtualenv) horstmann@linus:~$ python test.py 
base_prefix:  /opt/Bazarr/virtualenv
real_prefix:  /usr
sys.prefix:  /opt/Bazarr/virtualenv
True

```

My solution was to create a new variable that gathers sys.prefix and real_prefix and compare with base_prefix

```
    base_prefix = getattr(sys, "base_prefix", None)
    real_prefix = getattr(sys, "real_prefix", None) or sys.prefix
   return base_prefix != real_prefix

```

**Comment by horstmannmat** [comment]
I can do a merge request if needed

**Comment by morpheus65535** [comment]
Thanks! Yes please open a PR to dev branch. If you can do it today I'll merge it before we release 0.9.9.

**Comment by horstmannmat** [comment]
Done :)

**Comment by morpheus65535** [comment]
Fix by #1536

**Comment by morpheus65535** [comment]
Thanks!
